src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -398,6 +408,41 @@ public MessageActionImpl setActionRows(@Nonnull ActionRow... rows)
         return this;
     }
 
+    @Nonnull
+    @Override
+    public MessageAction setStickers(@Nullable Collection<? extends StickerSnowflake> stickers)
+    {
+        if (isEdit())
+            throw new IllegalStateException("Cannot edit stickers on messages!");
+        if (stickers == null || stickers.isEmpty())
+        {
+            this.stickers = new ArrayList<>();
+            return this;
+        }
```

Comment:
Not sure if it matters, but setting this to an empty array list instead of `null` will cause `sticker_ids: []` to be sent.

Reply:
Yes, that is intended. This would be the logical way to handle it if PATCH was supported for stickers. In the future, if they allow edit we only need to remove the check and it works. For the POST endpoint, empty, null, and omitted are treated the same way by discord.